commit e669e6b23e85d6f8f27a1b03cb0434a624291c1a (refs/changes/87/4987/3)
Author: Dave Eddy <dave@daveeddy.com>
Date:   2018-10-23T12:43:53-04:00 (12 months ago)
    
    TRITON-895 want more context around vmadm failures during smartdc/init
    Reviewed by: Pedro Palazón Candel <pedro@joyent.com>
    Approved by: Pedro Palazón Candel <pedro@joyent.com>

diff --git a/scripts/headnode.sh b/scripts/headnode.sh
index 08c1140f..a5431b1d 100755
--- a/scripts/headnode.sh
+++ b/scripts/headnode.sh
@@ -6,7 +6,7 @@
 #
 
 #
-# Copyright 2017 Joyent, Inc.
+# Copyright 2018 Joyent, Inc.
 #
 
 #
@@ -530,7 +530,7 @@ function create_zone {
         > ${payload_file}
     fi
 
-    cat ${payload_file} | vmadm create
+    VMADM_DEBUG_LEVEL='debug' vmadm create -f "$payload_file"
 
     local loops=
     local zonepath=
